interfaces/Portalicious/src/app/components/form/form-sidebar.component.html

```diff
@@ -9,10 +9,13 @@
   (onHide)="formGroup().reset()"
 >
   <ng-template pTemplate="headless">
+    <!-- eslint-disable-next-line @angular-eslint/template/interactive-supports-focus -->
     <form
       class="flex min-h-full w-full flex-col p-4"
       [formGroup]="formGroup()"
       (ngSubmit)="onFormSubmit()"
+      (keydown.control.enter)="onFormSubmit()"
+      (keydown.meta.enter)="onFormSubmit()"
```

Comment:
Hmm.. would it maybe be necessary to use a little key-event-specific wrapper-method, in which we could check whether this `form` also currently has focus within (or simply if its visible)?

(Hmm. maybe the angular-logic will already check for whether this key-event is triggered from 'inside' this form?)

To prevent false-positives when there are maybe multiple `form`-elements still available in the DOM at the same time?

Reply:
> (Hmm. maybe the angular-logic will already check for whether this key-event is triggered from 'inside' this form?)

Indeed, it looks like that is the case (from my very limited testing)